### DISCUSSION TITLE
Control managed minikube resources

### DISCUSSION THREAD

**Discussion by badarsebard** [GoogleCloudPlatform/cloud-code-intellij#3156] [Q&A]
Is there a way to specify the CPU and memory allocated to the minikube cluster when using the `Deploy locally to a minikube cluster` deployment option? I know I can create my own cluster and then point to that context, but I like having Cloud Code manage it for me. The environment variables in the advanced settings section seem to only be for the skaffold execution and have no impact on the minikube cluster.

**Comment by etanshaul** [comment]
hi @badarsebard this ability does not exist today. We do recommend for more advanced cases to create and manage your own cluster and point to it (exactly like you suggested). Part of our reasoning around this is that this is generally more reliable - having Cloud Code keep up with the configurations options of the myriad of tools it supports can be error prone and can go stale quickly as the tools change.
